index.html

```diff
@@ -1813,6 +1861,21 @@ <h3><dfn>constructor()</dfn> method</h3>
         <li>
           Perform the following steps [=in parallel=].
           <ol>
+            <li>
+              If the requested |localPort| is less than 32678, [=queue a global task=] on the [=relevant global object=] of
```

Comment:
What is the reason behind this? I could see requiring a local port higher than 1024, but 32,678 seems overly restrictive.

Reply:
Discussed this offline -- this is a security requirement for the launch.